### ISSUE TITLE
Problem running go get according to instructions on macOS monterey, with m1 chip

### ISSUE THREAD

**Issue by ajzo90** [marcboeker/go-duckdb#10]
Failing to install this package. Perhaps someone can guide me how to get up and running? 

Installed duckdb and verified that it works. 

Get a different error when passing in the environment variables, guessing it's because version mismatch? Using latest (Mar 4, 2022) commit from duckdb master branch `8d8a8d16dee075984c847d9e81cd0e2d7539f1c7`. 

Thanks! 

```
>> go get github.com/marcboeker/go-duckdb
# github.com/marcboeker/go-duckdb
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/connection.go:4:10: fatal error: 'duckdb.h' file not found
#include <duckdb.h>
         ^~~~~~~~~~
1 error generated.
```

```
>> CGO_LDFLAGS="-L/Users/christianpersson/repos/duckdb/build/release/src" CGO_CFLAGS="-I/Users/christianpersson/repos/duckdb/src/include" DYLD_LIBRARY_PATH="/Users/christianpersson/repos/duckdb/build/release/src" go get github.com/marcboeker/go-duckdb

# github.com/marcboeker/go-duckdb
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/connection.go:105:40: res.error_message undefined (type _Ctype_struct___0 has no field or method error_message)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:28:34: r.r.column_count undefined (type *_Ctype_struct___0 has no field or method column_count)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:30:25: c.name undefined (type _Ctype_struct___1 has no field or method name)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:41:26: r.r.row_count undefined (type *_Ctype_struct___0 has no field or method row_count)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:47:25: r.r.column_count undefined (type *_Ctype_struct___0 has no field or method column_count)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:50:13: col._type undefined (type _Ctype_struct___1 has no field or method _type)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:54:48: col.data undefined (type _Ctype_struct___1 has no field or method data)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:56:48: col.data undefined (type _Ctype_struct___1 has no field or method data)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:58:49: col.data undefined (type _Ctype_struct___1 has no field or method data)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:60:49: col.data undefined (type _Ctype_struct___1 has no field or method data)
../../go/pkg/mod/github.com/marcboeker/go-duckdb@v0.0.0-20211122095031-ceaaaf0726f1/rows.go:60:49: too many errors
```

**Comment by marcboeker** [comment]
Hi, DuckDB has changed some internals starting in version 0.3.2. I've added a fix (#11) for this and merged it. Could you please try it again?

**Comment by ajzo90** [comment]
Wow, awesome! Thanks a lot!
